caluma/form/validators.py

```diff
@@ -157,18 +152,21 @@ def validate(self, *, question, document, info, **kwargs):
         # Check all possible fields for value
         answer_tree = kwargs.pop("answer_tree", {})
         value = None
-        for i in ["value", "file", "date", "documents", "value_document"]:
+        for i in ["value", "file", "date", "documents"]:
             value = kwargs.get(i, value)
             if value:
                 break
 
         # empty values are allowed
         # required check will be done in DocumentValidator
         if value:
-            validate_func = getattr(self, f"_validate_question_{question.type}")
-            validate_func(
-                question, value, document=document, info=info, answer_tree=answer_tree
-            )
+            if not isinstance(answer_tree, list):
+                answer_tree = [answer_tree]
+            for at in answer_tree:
```

Comment:
I'm not sure what happens here: Why is `answre_tree` a list? Is this to validate in context of table questions?

If I understand correctly, you'd get `documents` in `kwargs` and would need to iterate over those, not multiple answer trees, no?

Reply:
Thanks for pointing this out! I've meant to come back to this and forgot about it.